### ISSUE TITLE
Skyline Error: The document does not contain any peptides.

### ISSUE THREAD

**Issue by azying312** [Nesvilab/FragPipe#1654]
Hi Fengchao, 

I am having an error in the Skyline document generation where it says there are no peptides found. I have attached the log file. The zipped file with the interact files is quite large, so I was wondering if it would be possible to share directly with either you or a lab member directly through Google Drive instead?

[log_2024-06-27_18-32-54.txt](https://github.com/user-attachments/files/16037103/log_2024-06-27_18-32-54.txt)

Best,
Alice

**Comment by fcyu** [comment]
Hi Alice,

Could you upload the files to https://www.dropbox.com/request/5dcG36IF6nHFhlJzxlbZ ?

Thanks,

Fengchao

**Comment by azying312** [comment]
I've uploaded the files, thank you!

**Comment by fcyu** [comment]
Thanks, but the pep.xml files are not enough to reproduce the error. I need the raw file, fasta file, and all files in the result folder.

Best,

Fengchao

**Comment by azying312** [comment]
The information should all be uploaded now!

Best,
Alice

**Comment by azying312** [comment]
Hi Fengchao,

I can run the Skyline module now too, it's just the the expected number of peptides and proteins is much less than I expect, but this might be more related to the issue #1647 and the peptide alignment issue.

Best,
Alice

**Comment by azying312** [comment]
Hi Fengchao,

I reran the analysis adding the biognosys iRT peptides to the fasta file, but I've been getting the problem again with the peptides. I am attaching the new log file and I will reupload the raw files, fasta file, and results files.

[log_2024-07-03_12-47-08.txt](https://github.com/user-attachments/files/16090209/log_2024-07-03_12-47-08.txt)

Best,
Alice

**Comment by fcyu** [comment]
I have no idea about your error since I couldn't reproduce it.

Could you move all of your files to local drive, upgrade Skyline-daily to the latest version, and try again?

Thanks,

Fengchao

**Comment by azying312** [comment]
Hi Fengchao,

I am rerunning on the new version of Skyline and changed the input from speclib to pepxml. As a clarification, why were speclib files not found after running through the specLib module? I didn't see an area to configure the output for the module, so I also wasn't sure what to select for the Skyline running mode. I've changed it to pep.xml now.

Best,
Alice

**Comment by fcyu** [comment]
> As a clarification, why were speclib files not found after running through the specLib module?

Could you upload the log file? But I guess that was because you didn't enable DIA-NN.

Best,

Fengchao

**Comment by azying312** [comment]
Here's the log file [log_2024-07-03_12-47-08.txt](https://github.com/user-attachments/files/16090944/log_2024-07-03_12-47-08.txt), I didn't realize it was DIA-NN specific, thank you!

**Comment by fcyu** [comment]
You are welcome. As a beginner, I strongly suggest you do not change any parameters unless you really understand them.

Best,

Fengchao

**Comment by azying312** [comment]
Hi Fengchao,

Thank you, I really appreciate all the support. I used the Skyline running mode "Using the pep.xml as input", Skyline Daily, and left the special modifications mode on default. I've also updated to the newest version of Skyline Daily, however, the .blib files didn't generate at all this time. I am attaching my log file: [log_2024-07-03_16-41-57.txt](https://github.com/user-attachments/files/16091693/log_2024-07-03_16-41-57.txt) Between this run and the last run, the only parts I changed were that I updated Skyline Daily and I changed the running mode to use pep.xml instead of speclib since I am using the default workflow and not using DIA-NN. It seems like there weren't issues with parsing through the peptides.

I opened the generated library.tsv file on Skyline Daily on my own and it seems to have worked fine, so the only issue was from generating a .skys document.

Best,
Alice
